planemo/bioconda_scripts/bioconductor_skeleton.py

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/env python
+
```

Comment:
Yes, @jmchilton makes a good point. I was about to make a few edits to the script as well .i.e deleting the cached tarball if the user chooses.